templates/vue/src/components/Lightbox/media/MultiContentMedia/PageRows.vue

```diff
@@ -165,9 +165,9 @@ export default {
     },
     rowBackground() {
       if (this.isMultiContentContext) {
-        let rgb = 40
-        let colorOffset = this.level * 10
-        rgb = colorOffset > rgb ? 0 : rgb - colorOffset
+        let rgb = 187
+        // let colorOffset = this.level * 10
+        // rgb = colorOffset > rgb ? 0 : rgb - colorOffset

```

Comment:
Again, this functionality should not be removed. I suggest using the tinycolor2 functionality instead of what's in here.